js/tic_tac_toe.js

```diff
@@ -1,2 +1,80 @@
-// Your code here
-// Use the jQuery
+$(document).ready(function() {
+  $('.box_cell').on('click', TicTacToe.takeTurn);
+});
+
+
+var TicTacToe = (function(){
+  var turnCounter = 1;
+  var board = [
+    [0,0,0],
+    [0,0,0],
+    [0,0,0]
+  ]
+
+  var takeTurn = function() {
+    if (!$(this).hasClass('used')){
+        turnCounter % 2 === 0 ? redTurn($(this)) : blackTurn($(this));
+        $(this).addClass('used');
+        turnCounter++;
+        checkBoard();
+      }
+  };
+
+  var redTurn = function(cell){
```

Comment:
redTurn and blackTurn feel incredibly similar. How could you make one function to do both?